_data/accessibility-tests/file-input.yml

```diff
@@ -0,0 +1,103 @@
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.3
+  - summary: File input elements have proper contrast.
+    summary_additional: When you use a color contrast analyzer, the contrast between the file input and all of its elements is at least 3:1 from their respective surrounding backgrounds.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.11
+  - summary: Error text indicates an incorrect file type.
+    summary_additional: When you drag an incorrect file type into the file upload box, text appears describing the error.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 3.3.1
+  - summary: Instructions for file type and use are present.
+    summary_additional: When you view a file input box, you see clear, brief instructions for selecting and uploading the correct file type.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 3.3.2
+# Zoom/screen magnification tests
+  - summary: File input box is visible and functional when screen is magnified.
+    summary_additional: When you zoom in to 200%, the file input box and content remain visible and retain functionality.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.4
+# Keyboard navigation tests
+  - summary: Focus indicator is clearly visible.
+    summary_additional: When you use a keyboard to focus on a file input box you will see a visible outline or other clear indication of focus.
+    test_status: exception
+    status_details: Focus indicator is visible around the "Drag files" area as well as the "choose from folder" action link.
+    github_issue_number: 5616
+    github_issue_repo: uswds
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.7
+  - summary: File can be input with only a keyboard.
+    summary_additional: You can successfully upload a file using only a keyboard. No mouse or dragging action is required.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.5.7
+  - summary: Content doesn't change until the user takes an action to change it.
+    summary_additional: When you access the file input with a keyboard, content stays the same until you use keys to make changes.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.2
+# Screen reader tests
+  - summary: Screen reader announces file input.
+    summary_additional: Screen reader announces file input. When you navigate with arrow keys and the tab key while using a screen reader, you hear the prompt to “drag file here or choose from folder” and you hear any related hints, instructions, or errors.
+    test_status: exception
+    status_details: The file input itself is announced correctly however the error when uploading an incorrect file type is not.
```

Comment:
**suggestion:** Recommend we get more specific about what is working and what isn't to improve clarity.

```suggestion
    status_details: The file input appropriately announces the component instructions and hints to the screen reader. However, it does not announce an error message when an invalid file type is added.
```

Reply:
Resolved in [a293494](https://github.com/uswds/uswds-site/pull/2963/commits/a293494611d9afbfb645af8e11a55592761789cf)